tests/by_image/base-notebook/test_container_options.py

```diff
@@ -15,13 +15,13 @@
 def test_cli_args(container: TrackedContainer, http_client: requests.Session) -> None:
     """Image should respect command line args (e.g., disabling token security)"""
     host_port = find_free_port()
-    running_container = container.run_detached(
```

Comment:
There is no need to return anything from `run_detached` - TrackedContainer has a container inside and it's better to provide specific functions we need (this way we're encapsulating docker functions we use)